src/utils.ts

```diff
@@ -272,11 +272,16 @@ export async function checkIfOnline() {
 }
 
 /**
- * Saves the script ID in the project dotfile.
+ * Saves the script ID, rootDir in the project dotfile.
  * @param  {string} scriptId The script ID
+ * @param  {string} rootDir Local root directory that store your project files
  */
-export async function saveProjectId(scriptId: string): Promise<string> {
-  return DOTFILE.PROJECT().write({ scriptId }); // Save the script id
+export async function saveProject(scriptId: string, rootDir?: string): Promise<ProjectSettings> {
+  const project: ProjectSettings = { scriptId };
+  if (rootDir) {
+    project.rootDir = rootDir;
+  }
```

Comment:
It's common to write this equivalent statement:
```ts
project.rootDir = project.rootDir || rootDir;
```
instead of
```ts
if (rootDir) {
  project.rootDir = rootDir;
}
```
Can you replace this with the shorter line of code?